components/cell/jwks-server/crypto/resolver/file_based_resolver.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package resolver
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"log"
+)
+
+const KeyFilePath string = "/etc/certs/key.pem"
+const CertFilePath string = "/etc/certs/cert.pem"
+
+func FileBasedKeyResolver(privateKeyStr []byte, certificateStr []byte) error{
+	blockPriv, _ := pem.Decode([]byte(privateKeyStr))
+	//To get RSA Private key
+	key, errKey := x509.ParsePKCS1PrivateKey(blockPriv.Bytes)
```

Comment:
Please update all error handling variables to "err" and reuse for consistency. Please fix in other places as well.

Reply:
pem.decode does not returns an error 
func Decode(data []byte) (p *Block, rest []byte) {

I have fixed the other places